Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -1662,6 +1667,7 @@ void AODProducerWorkflowDPL::run(ProcessingContext& pc)
   // helper map for fast search of a corresponding class mask for a bc
   std::unordered_map<uint64_t, uint64_t> bcToClassMask;
   if (mInputSources[GID::CTP]) {
+    LOG(info) << "CTP input available";
```

Comment:
These logs at every TF is also too much, printing at 1st TF only should be enough.

Reply:
info changed to debug